docs/release-notes/october-2024.md

```diff
@@ -0,0 +1,130 @@
+
+### Changed
+
+- Various UI improvements to the **Settings > SCM** tab.
+![Old SCM card](/img/old-scm-card.png) 
+![Updated SCM card](/img/new-scm-card.png)
+_**Figure**. Previous and current SCM card UI._
+- **Semgrep Managed Scans**: scans now follow fail open behavior, consistent with how Semgrep in CI behaves. 
+- The **Projects** page's **See findings** button is now a drop-down box, enabling you to select which product you want to view findings for.
+
+### Fixed
+
+- When a scan runs into an exception, Semgrep AppSec Platform is appropriately notified about the failure. Previously, within the AppSec Platform UI, it would appear to the user that the scan is still in progress.
+- Fixed a bug where Semgrep would crash if `--trace` was passed.
+
+## 💻 Semgrep Code
+
+### Added
+
+- Updated the C# parser to support all versions of the language up to 13.0 (.NET 9).
+- Developers can now triage findings by replying to a GitHub PR comment from Semgrep, without the need to log in to Semgrep Cloud Platform. See [Triage findings through comments](/semgrep-code/triage-remediation#triage-findings-through-pr-and-mr-comments) for more information.
+- Added an API endpoint you can use to triage findings in bulk, either by passing a list of `issue_ids` or filter query parameters to select findings. Refer to [<i class="fas fa-external-link fa-xs"></i> Bulk triage API documentation](https://semgrep.dev/api/v1/docs/#tag/TriageService).
+- Taint analysis now supports tracking sinks through callbacks for all applicable Semgrep-supported languages. For example:
+
+```javascript
+function unsafe_callback(x) {
+  sink(x); // Semgrep detects a finding here now!
+}
+
+function withCallback(val, callback) {
+  callback(val);
+}
+
+withCallback(taint, unsafe_callback)
+```
+
+### Removed
+
+- Removed support for Vue. The `tree-sitter grammar` has not been updated in 3 years and no community rules have been added. In theory, extract mode could be a good substitute to parse Vue files.
+
+## ⛓️ Semgrep Supply Chain
+
+### Added
+
+- Supply Chain now provides reachability analysis for Kotlin, including support for Gradle and Maven.
+- Improved support and flexibility to Python dependency parsing (public beta):
+  - Semgrep now finds non-standard `requirements.txt` names and parses them for dependencies. 
+  - Semgrep parses lockfiles in a `requirements` folder.
+- `cargo.lock` parser can now associate dependencies with lockfile line numbers.
+
+### Changed
+
+- Improvements to the **Advisories** page UI. <!-- 16657 -->
+- **Dependency search**: The **Ecosystem** filter has been replaced by a **Language** filter. Several languages can share the same ecosystem, such as Java and Kotlin both using Maven. For accurate filtering, the Dependencies page now uses a Language filter so that you can view that language's packages from any ecosystem supported by Semgrep for that language.
```

Comment:
```suggestion
- **Dependency search**: The **Ecosystem** filter has been replaced by a **Language** filter. Several languages can share the same ecosystem, such as Java and Kotlin both using Maven. For accurate filtering, the **Dependencies** page now uses a **Language** filter so that you can view that language's packages from any ecosystem supported by Semgrep for that language.
```